### DISCUSSION TITLE
NAS library not working after mpd database creation

### DISCUSSION THREAD

**Discussion by ben3595** [rern/rAudio#1322] [General]
Hi, I'm a new user, just started using raudio yesterday. 

After several hours waiting for the library to create (I can see the files being scanned in the mpd information screen) nothing is added to the library.

Player screen says Database 0 but /srv/http/data/mpd/mpd.db is 3.1mb. If I open the NAS option in Library the message is 'Database not yet available...' 

Am I doing something wrong?

**Comment by ceejayemm** [comment]
If the music files in your library are encrypted in some way (mine are *.m4a for example) you will need to turn on the Player / Options / FFmpeg option and then do a Full Database update in order to load them into your database. Click the '?' in Player / Options then again on the FFmpeg setting to see the list of audio file types this option supports.

Chris

**Reply by ben3595** [reply]
Thanks for your advice Chris, but it didn't work. I can see ffmpeg plug-in enabled in mpd, ran a full database rebuild, still no library.

**Reply by rern** [reply]
Try:
- Disconnect NAS
- Copy an album to `/mnt/MPD/SD`
- Update Library database

**Reply by ben3595** [reply]
Yes that worked, so I double checked the share permissions on my NAS. There's 3 different places for nfs permissions on Synology. I missed the Advanced Permissions for the Guest account.
